pdr_backend/lake/etl.py

```diff
@@ -314,10 +314,10 @@ def update_bronze_pdr(self):
         """
         logger.info("update_bronze_pdr - Update bronze tables.")
 
-        # st_timestamp and fin_timestamp should be valid UnixTimeMS
-        st_timestamp, fin_timestamp = self._calc_bronze_start_end_ts()
-
         for table_name, get_data_func in self.bronze_table_getters.items():
+            # st_timestamp and fin_timestamp should be valid UnixTimeMS
+            st_timestamp, fin_timestamp = self._calc_bronze_start_end_ts(table_name)
```

Comment:
This is wrong.
1. All tables should have already been processed up-to-the-same end-ts
2. We do not need to calculate st_ts and en_ts for each table.
3. We can get the max(table1, table2, table3) across all raw_tables as end_ts
4. And the max(table1, table2) across all bronze_table as st_ts